components/org.wso2.carbon.identity.branding.preference.resolver/src/main/java/org/wso2/carbon/identity/branding/preference/resolver/UIBrandingPreferenceResolverImpl.java

```diff
@@ -855,6 +868,41 @@ private ConfigurationManager getConfigurationManager() {
         return BrandingResolverComponentDataHolder.getInstance().getConfigurationManager();
     }
 
+    private void getCustomContentIfNeeded(ObjectNode objectRoot, JsonNode root, String resolvedSourceName, String type)
+            throws CustomContentServerException {
+
+        String activeLayout = root.path(LAYOUT_KEY).path(ACTIVE_LAYOUT_KEY).asText();
+        if (!CUSTOM_CONTENT_TYPE.equals(activeLayout)) {
+            objectRoot.remove(CUSTOM_CONTENT_KEY);
+            return;
+        }
+
+        CustomContent customContent = null;
+
+        if (type.equals(APPLICATION_TYPE)) {
```

Comment:
This condition check is not null safe. Better to have the constant at the first, so that it won't throw any NPE even though the value of `type` is null. 
Check other places as well